src/pages/oasis.js

```diff
@@ -0,0 +1,197 @@
+import React from "react"
+import styled from "styled-components"
+import Layout from "components/PageLayout/layout"
+import SEO from "components/seo"
+import Banner from "styles/components/Banner"
+import tree from "images/oasis2.svg"
+
+import {
+  SB_NAVY,
+  SB_ORANGE,
+  SB_SALMON,
+  SB_LIGHT_ORANGE,
+  SB_LIGHT_SALMON,
+} from "@colors"
+
+const Title = styled.div`
+  color: ${SB_ORANGE};
+  display: flex;
+  font-family: Montserrat;
+  font-weight: 600;
+  font-size: 50px;
+  font-style: italic;
+  text-transform: uppercase;
+  margin: 80px auto;
+  margin-bottom: 40px;
+  width: 200px;
+  @media (max-width: 600px) {
+    justify-content: center;
+    padding: 0;
+  }
+`
+
+const SmallTitle = styled(Title)`
+  font-size: 30px;
+  width: 300px;
+  justify-content: center;
+  margin-bottom: 40px;
+`
+
+const Text = styled.span`
+  color: ${SB_NAVY};
+  font-weight: 400;
+  font-size: 28px;
+  font-style: italic;
+  font-family: Montserrat;
+  margin: 40px auto;
+  margin-bottom: 160px;
+  display: block;
+  width: 50%;
+  @media (max-width: 600px) {
+    font-size: 20px;
+  }
```

Comment:
I noticed that the text on the [Sandbox about page](https://sandboxneucom-git-oasispage-sandboxneu.vercel.app/about) is wider than this, do you think it makes sense to make them both the same?

Either way I do think it would make it read better if the width increased slightly, not sure how much is would be best though.

Reply:
yup I can increase it